docs/backend/guides/how-to/soft-delete.md

```diff
@@ -26,16 +26,59 @@ type ExampleModel struct {
 
 If this has not been done, one must [create a migration](migrate-the-database.md) to make these changes.
 
-Furthermore, any queries to fetch the model must exclude those that have been 'soft deleted'.
+## Querying for non-deleted records
+
+Records that have been soft deleted will still exist in the database, so we must filter them out in our database queries if we want to omit deleted data.
+
+The Pop ORM has a chain-able method called [Scope](https://gobuffalo.io/documentation/database/scoping/) that we can use to append the where clause(s) to only include non-deleted records.
 
 ```go
-func FetchExampleModel(ctx context.Context, db *pop.Connection, session * auth.Session, id uuid.UUID) (ExampleModel, error) {
-    var exampleModel ExampleModel
-    err := db.Q().Where("example_models.deleted_at is null").Eager().Find(&exampleModel, id)
-    ...
+func FindShipment(ctx context.Context, shipmentID uuid.UUID) {
+    var shipments models.MTOShipments
+    ctx.DB().Scope(utilities.ExcludeDeletedScope()).All(&shipments)
 }
 ```
 
+Sometimes you will need to qualify the deleted_at column with the model(s) that you care about to avoid an ambiguous column SQL error. You can achieve this by passing in the models themselves to the ExcludeDeletedScope method. ExcludeDeletedScope will look up the proper table name of the model, using either reflection or the TableName() override specified in the model file.
+
+```go
+func FindDocumentsWithUploads(ctx context.Context, uploaderID uuid.UUID) {
+    var documents models.Documents
+    ctx.DB().Scope(utilities.ExcludeDeletedScope(models.Document{}, models.UserUpload{})).
+        Join("user_uploads", "user_uploads.document_id = documents.id").
+        All(&documents)
+}
+```
+
+:::caution
+
+Unfortunately this will not filter any eager loaded associations so you may need to continue doing that depending on your query.
```

Comment:
Nit: `continue doing that` is a bit vague. I assume this is referring to filtering, but we should be explicit here.

Reply:
Mmm yea that assumes an instance where you were already filtering after the fact as opposed to a new case.